media-api/app/lib/elasticsearch/TogglingElasticSearch.scala

```diff
@@ -3,24 +3,16 @@ package lib.elasticsearch
 import com.gu.mediaservice.lib.auth.Authentication
 import com.gu.mediaservice.model.Image
 import lib.SupplierUsageSummary
-import play.api.Logger
 import play.api.mvc.{AnyContent, Security}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class TogglingElasticSearch(a: ElasticSearchVersion,
                             b: ElasticSearchVersion) extends ElasticSearchVersion {
 
-  val OPT_OUT_COOKIE_NAME = "GRID_ELASTIC6_OPT_OUT"
-
-  def active()(implicit request: Security.AuthenticatedRequest[AnyContent, Authentication.Principal]) = {
-    val userIdentifier = request.user.apiKey.name
-    val userHasRequestedOptOut = request.cookies.exists(c => c.name == OPT_OUT_COOKIE_NAME)
-    if (userHasRequestedOptOut) {
-      Logger.info("User " + userIdentifier + " is still opted out of Elastic 6 indexes")
-      a
-    } else b
-  }
+  // `a` is ES1, `b` is ES6... always use ES6
```

Comment:
Lets rename the variables for clarity

Reply:
This changes in https://github.com/guardian/grid/pull/2530. I'll ship this then rebase that PR to get this rename in.